PermissionsEx/permissions.yml terminatorgenisys

```diff
@@ -595,6 +595,30 @@ groups:
         - ch.alias.haste4
         - ch.alias.haste5
         - ch.alias.haste6
+terminatorgenisys:
```

Comment:
https://video.search.yahoo.com/yhs/search?fr=yhs-omr-001&hsimp=yhs-001&hspart=omr&p=the+terminator+beating+up+bill+paxton#id=1&vid=bda2c7df659042b251ae8ef4a2445911&action=click